azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: $(SourceBranchName)-$(Date:yyyyMMdd)$(Rev:.rrr)
+trigger:
+  batch: false
+  branches:
+    include:
+    - master
+    - release-*
+
+variables:
+  branch: $[ coalesce(variables['system.PullRequest.TargetBranch'], variables['build.SourceBranchName']) ]
+
+pool:
+  vmImage: ubuntu-16.04
+container:
+  image: sykesm/fabric-protos:0.1
```

Comment:
Debian-based Go build env?  Does this eventually go under hyperledger org somewhere?

Reply:
The Dockerfile is in this repo but we don't have anything setup to build and push updates there. We'll need to do that eventually.